app/assets/stylesheets/components/_pagination.scss

```diff
@@ -8,7 +8,7 @@
   display: flex;
   justify-content: space-between;
   flex-direction: row-reverse;
-  align-items: flex-end;
+  align-items: center;
```

Comment:
Vertical align the text "Displaying clients 1 - 25 of 95 in total" with the bottom of the button looks weird - it should be center aligned.